nuxt.config.js

```diff
@@ -23,4 +24,99 @@ c.dir = {
   store:      path.posix.join(SHELL, 'store'),
 };
 
+c.build.extend = (config, { isClient, isDev }) => {
+  // Override the webpack target, so that we get the correct mix of
+  // electron (chrome) + nodejs modules (for ipcRenderer).
+  config.target = 'electron-renderer';
+  // Set a resolver alias for `./@pkg` so that we can load things from @ in CSS
+  config.resolve.alias['./@pkg'] = path.resolve(__dirname, 'pkg', 'rancher-desktop');
+
+  if ( isDev ) {
+    config.devtool = 'cheap-module-source-map';
+  } else {
+    config.devtool = 'source-map';
+  }
+
+  // if ( resourceBase ) {
+  //   config.output.publicPath = resourceBase;
+  // }
+
+  // Remove default image handling rules
```

Comment:
Do you recall _why_ we needed to remove the default image handling rules?

Reply:
We're looking at a lot of `@rancher/shell` configuration details here in the root-level `nuxt.config`. I can try to sync up with the authors to get more information but we have this information exposed because we need to extend the existing `@rancher/shell` config (imported on line 7: `import config from '@rancher/shell/nuxt.config';`) in order to allow the shell to run properly in an electron environment.